src/components/CarouselCard/index.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-script-url,max-len */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Provider } from './context';
+import Indicators from './indicators';
+import AnimationButton from './animationButton';
+import { getSelectedItemIndex, getChildTabNodes, insertChildOrderly } from './utils';
+import './styles.css';
+
+export default class CarouselCard extends Component {
+    constructor(props) {
+        super(props);
+        this.registerChild = this.registerChild.bind(this);
+        this.setActiveItem = this.setActiveItem.bind(this);
+        this.handleOnClick = this.handleOnClick.bind(this);
+        this.state = {
+            childrenRegistred: [],
+            activeItem: undefined,
+            stopAnimation: this.props.disableAutoScroll,
```

Comment:
Let's call this isAnimationPaused since it is a boolean value, stopAnimation is an action it is better for a function name.